symfony/plugins/orangehrmRecruitmentPlugin/test/orangehrmRecruitmentPluginAllTests.php

```diff
@@ -7,7 +7,7 @@ public static function suite() {
 
         /* Dao Test Cases */
         $suite->addTestFile(dirname(__FILE__) . '/model/dao/CandidateDaoTest.php');
-        $suite->addTestFile(dirname(__FILE__) . '/model/dao/VacancyDaoTest.php');
+        $suite->addTestFile(dirname(__FILE__) . '/Dao/VacancyDaoTest.php');
```

Comment:
no need to register here. we are going to remove this file.
use `phpunit.xml` to define wildcard paths